### DISCUSSION TITLE
vsg/core/Version.h issue

### DISCUSSION THREAD

**Discussion by joseluispascual** [vsg-dev/VulkanSceneGraph#1286] [Q&A]
I’ve successfully installed my VSG libraries, but I’m encountering a compilation error when I try to build my project. The error states that it cannot find /core/Version.h in the include folder. Initially, I suspected that the installation might have gone wrong. However, upon checking the repository, I discovered that the file is present, but it’s located in the src folder alongside Version.cpp. Has anyone else experienced this issue? Any insights would be greatly appreciated. Thanks!

**Comment by robertosfield** [comment]
The CMake build system provides and install target that you should run
after a build to place all the libs, CMake config files and headers in the
CMake install path location.

**Reply by joseluispascual** [reply]
Already fixed! Thanks for everything :) !

**Reply by robertosfield** [reply]
I'm curious what you original did. Did you not run the install target? Did you manually copy files or just set paths to the original build directories?
